client/src/pages/RecipePage.jsx

```diff
@@ -1,7 +1,75 @@
-import React from "react";
+import React, { useState } from "react";
+import { Col, Row, Input, Pagination } from 'antd';
+
+import '../styles/RecipePage.less';
+import SelectionCard from '../components/SelectionCard';
+// TODO - get recipe data using API
+import recipeData from '../data/dummyRecipes.json'
+
+// I was going off of the figma diagram when choosing how many recipes to show
+const PAGE_SIZE = 6;
+const pageArray = (arr, pageNum, size) => {
+  return arr.slice((pageNum - 1) * size, pageNum * size);
+}
+
+const mapRows = (recipeArr) => {
```

Comment:
I think you can actually do this by just mapping the result json and return a `Col` object inside a single row where each column only takes up half the row so that each "row" will only consist of 2 columns at most.

BUTTTTTT this also works perfectly fine :))) so no need to change anything. Just a suggestion :)))